### ISSUE TITLE
508-defect-1 ❗️ Launchblocker: Address of location to register in person missing on register for care

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#21350]
Label: 20132

    - [508-defect-1 :exclamation: Launchblocker]

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1

Issue Title: Register for care, address of location to register in person missing.
 ## Point of Contact: Angela Fowler
**VFS Point of Contact:** _First name only: Angela

<h2>User Story or Problem Statement:</h2>
As a person with a disability who depends on public transportation, it is essential that I have the exact address of my destination when booking rideshare or para transit, or planning a trip using the city bus.
<h2>Details</h2>
Under the level 2 heading Patient registration (admissions), the user is given the option to register for care by phone or in person. It says that one can register for care at the eligibility benefits office, ground floor, near main entrence. It gives no address, nor city or state. 
Acceptance Criteria

Environment

Steps to Recreate

Proposed Solution (if known)

<h2>WCAG or Vendor Guidance (optional)</h2>

https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-understanding.html

<h2>Screenshots or Trace Logs</h2>

![image](https://user-images.githubusercontent.com/78870095/110683467-e90ab980-8190-11eb-9435-3df48b63ccf2.png)
